mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
+Synapse ID of GeoMx DSP ROI Segment Annotation File,Synapse ID(s) for ROI/AOI/Segmentation annotations in the GeoMx DSP experiment.,,,FALSE,,,,HTAN,
+GeoMx DSP Assay Type,The assay type which was used for the GeoMx DSP pipeline.,,,FALSE,,,,HTAN,
+Target Depth,The targeted read depth prior to sequencing. Integer,,,TRUE,,,,HTAN,
+Read Length,"The length of the sequencing reads. Can be integer, null",,,TRUE,,,,HTAN,
+ROI Name,Name of Region of Interest corresponding to file. Multiple names should be comma-separated,,,TRUE,,,,HTAN,
+AOI Name,Name of Area of Illumination corresponding to file. Multiple names should be comma-separated,,,TRUE,,,,HTAN,
+Software and Version,Name of software used to generate expression values. String,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Sequencing Platform,A platform is an object aggregate that is the set of instruments and software needed to perform a process [OBI_0000050]. Specific model of the sequencing instrument.,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Library Selection Method,How RNA molecules are isolated.,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Library Preparation Kit Name,Name of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+GeoMx DSP Library Preparation Kit Vendor,Vendor of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+GeoMx DSP Library Preparation Kit Version,Version of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+NanoString GeoMx DSP Spatial Transcriptomics Level 2,Raw counts (DCC/RCC) files from GeoMX experiments,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of Parent FASTQ File, Synapse ID of GeoMx Lab Worksheet File, Synapse ID of GeoMx DSP PKC File, Synapse ID of GeoMx DSP ROI Segment Annotation File, GeoMx DSP Assay Type, ROI Name, AOI Name, Software and Version, GeoMx DSP NGS Sequencing Platform, GeoMx DSP NGS Library Selection Method, GeoMx DSP NGS Library Preparation Kit Name, GeoMx DSP Library Preparation Kit Vendor, GeoMx DSP Library Preparation Kit Version",FALSE,,,,HTAN,
+Synapse ID of Parent FASTQ File,Synapse ID(s) indicating the FASTQ file(s) from which the DCC file was derived; multiple parent files should be comma-separated,,,FALSE,,,,HTAN,
+NanoString GeoMx DSP Imaging Level 2,,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of GeoMx DCC File, Synapse ID of GeoMx ROI Segment Annotation File, ROI Name, AOI Name, Channel Metadata Filename, Imaging Assay Type, Protocol Link, Software and Version, Microscope, Objective, NominalMagnification, LensNA, WorkingDistance, WorkingDistanceUnit, Immersion, Pyramid, Zstack, Tseries, FOV number, FOVX, FOVXUnit, FOVY, FOVYUnit, Frame Averaging, Image ID, DimensionOrder, PhysicalSizeX, PhysicalSizeXUnit, PhysicalSizeY, PhysicalSizeYUnit, PhysicalSizeZ, PhysicalSizeZUnit, Pixels BigEndian, PlaneCount, SizeC, SizeT, SizeX, SizeY, SizeZ, PixelType",FALSE,,,,HTAN,
```

Comment:
We had discussed removing some of these attributes previously, but maintaining them all seems like a good option if we want to ensure the GeoMX image files are compatible with data from other experiments.

Reply:
I added "Synapse ID of GeoMx DCC File" because I think it makes sense to reference DCC file(s) associated with the image, but I'm not completely sure if this is necessary.

"ROI Name" and "AOI Name" were added to help make it clear which regions in the ROI annotation file are associated with the image.